tests/linearizability/linearizability_test.go

```diff
@@ -301,23 +332,62 @@ func simulateTraffic(ctx context.Context, t *testing.T, clus *e2e.EtcdProcessClu
 
 			config.traffic.Run(ctx, clientId, c, limiter, ids, lm)
 			mux.Lock()
-			h = h.Merge(c.history.History)
+			h.Merge(c.history.History)
 			mux.Unlock()
 		}(c, i)
 	}
+
+	simulatePostFailpointTraffic(ctx, &wg, endpoints, config.clientCount, ids, h, &mux, config, limiter, lm)
+
 	wg.Wait()
 	endTime := time.Now()
 	operations := h.Operations()
-	t.Logf("Recorded %d operations", len(operations))
+	lg.Info("Recorded operations", zap.Int("num-of-operation", len(operations)))
 
 	qps := float64(len(operations)) / float64(endTime.Sub(startTime)) * float64(time.Second)
-	t.Logf("Average traffic: %f qps", qps)
+	lg.Info("Average traffic", zap.Float64("qps", qps))
 	if qps < config.minimalQPS {
 		t.Errorf("Requiring minimal %f qps for test results to be reliable, got %f qps", config.minimalQPS, qps)
 	}
 	return operations
 }
 
+func simulatePostFailpointTraffic(ctx context.Context, wg *sync.WaitGroup, endpoints []string, clientId int, ids identity.Provider, h *model.History, mux *sync.Mutex, config trafficConfig, limiter *rate.Limiter, lm identity.LeaseIdStorage) {
```

Comment:
Don't understand why you cannot incorporate this into normal traffic

Reply:
To incorporate this into normal traffic, a client with test user authorization has to be set up upfront. However, before the user is added to the cluster, client creation will fail. 

After a couple of failed attempts, I adopted this workaround. It's more deterministic to reliably reproduce on the impacted 3.5 versions.

Reply:
Please setup authorization in cluster setup.